src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
   margin: 0;
+
+  --hover-color: #00acdc;
+}
+
+a {
+  color: #000;
```

Comment:
The color for links is set to black (`#000`). Ensure there is enough contrast between the link color and the background for accessibility. Consider using a tool to check color contrast ratios.